src/quo2/components/dropdown.cljs

```diff
@@ -0,0 +1,150 @@
+                                                  :padding-right 6}
+                              :padding-with-no-icon {:padding-vertical 3
+                                                     :padding-horizontal 8}}}})
+(defn color-by-10
+  [color]
+  (colors/alpha color 0.6))
+
+(defn dropdown-comp [{:keys [icon open? text dd-height size disabled? dd-color border-color]}]
+  (let [dark? (colors/dark?)
+        {:keys [width height width-with-icon padding font icon-size]} (get-in sizes [size])
+        {:keys [padding-with-icon padding-with-no-icon]} padding
+        font-size (:font-size font)
+        spacing 4]
+    [rn/touchable-opacity (cond-> {:on-press (fn []
+                                               (if (swap! open? not)
+                                                 (apply-anim dd-height 120)
+                                                 (apply-anim dd-height 0)))
+                                   :style (cond-> (merge
+                                                   (if icon
+                                                     padding-with-icon
+                                                     padding-with-no-icon)
+                                                   {:width (if icon
+                                                             width-with-icon
+                                                             width)
+                                                    :height height
+                                                    :border-radius (if (= :small size)
+                                                                     8
+                                                                     10)
+                                                    :flex-direction :row
+                                                    :align-items :center
+                                                    :border-width 1
+                                                    :border-color (if @open?
+                                                                    border-color
+                                                                    (color-by-10 border-color))
+                                                    :background-color (if @open?
+                                                                        dd-color
+                                                                        (color-by-10 dd-color))})
+                                            disabled? (assoc :opacity 0.3))}
+                            disabled? (assoc :disabled true))
+     (when icon
+       [icons/icon icon {:no-color true
+                         :size 20
+                         :container-style {:margin-right spacing
+                                           :margin-top 1
+                                           :width icon-size
+                                           :height icon-size}}])
+     [text/text {:size font-size
```

Comment:
I've read comments in other PRs that we prefer to align keys, so it seems some parts of this namespace are misaligned.

I don't know if we've ever discussed or tried https://github.com/kkinnear/zprint. It's an incredibly flexible tool, written in Clojure for Clojure. I understand code style can be almost personal, but if everyone agrees on the very basic styles, like map keys alignment we could easily automate this in the future.

Reply:
Can you tell me an example of an alignment you think looks better ?

Reply:
Sure, my comment was specifically about misaligned map keys in this namespace, and I've seen other PRs where reviewers said we prefer to align map keys. So again, this is not necessarily my preference, but it seems the rest of team prefers this style. If you're under Emacs, you probably know you can call `C-c SPC` to auto indent map keys and let bindings.

For example:

```clojure
;; Misaligned map keys.
[icons/icon icon {:no-color true
                  :size 20
                  :container-style {:margin-right spacing
                                    :margin-top 1
                                    :width icon-size
                                    :height icon-size}}]

;; Aligned map keys.
[icons/icon icon {:no-color        true
                  :size            20
                  :container-style {:margin-right spacing
                                    :margin-top   1
                                    :width        icon-size
                                    :height       icon-size}}]
```

So, on top of this idea, I mentioned about zprint, a tool that can automate checks for some of our most agreeable code styles so that we can reduce the effort to review PRs for things that a machine can do for us. This is more like an idea that I would like to raise to the team, but I thought it was worth mentioning here for you.

Reply:
Used vertical alignment